python/lsst/ts/xml/data/sal_interfaces/MTM2/MTM2_Telemetry.xml

```diff
@@ -496,7 +496,7 @@
     </item>
     <item>
       <EFDB_Name>weight</EFDB_Name>
-      <Description>Total force error to support the mirror's weight based on the force projection.</Description>
+      <Description>Total excess mass to support the dynamic load and mirror's weight that bases on the force projection.</Description>
```

Comment:
Perharps better - improve forces field as well:

```
diff --git a/python/lsst/ts/xml/data/sal_interfaces/MTM2/MTM2_Telemetry.xml b/python/lsst/ts/xml/data/sal_interfaces/
index 7dd12a3..49996f0 100644
--- a/python/lsst/ts/xml/data/sal_interfaces/MTM2/MTM2_Telemetry.xml
+++ b/python/lsst/ts/xml/data/sal_interfaces/MTM2/MTM2_Telemetry.xml
@@ -489,14 +489,14 @@
     <Description>Tangential force error used to compare with the thresholds to avoid the breaking of mirror. A1 and
     <item>
       <EFDB_Name>force</EFDB_Name>
-      <Description>Force error to support the mirror's weight for individual tangential links.</Description>
+      <Description>Individual tangential link force excess. Calculated as difference between expected (calculated fr
       <IDL_Type>double</IDL_Type>
       <Units>newton</Units>
       <Count>6</Count>
     </item>
     <item>
       <EFDB_Name>weight</EFDB_Name>
-      <Description>Total force error to support the mirror's weight based on the force projection.</Description>
+      <Description>Total mirror tangential force excess. Calculated from individual force excess (difference between
       <IDL_Type>double</IDL_Type>
       <Units>newton</Units>
       <Count>1</Count>
```

Reply:
Improved in the commit: [964e320](https://github.com/lsst-ts/ts_xml/pull/780/commits/964e32020ce6e81fdd7325a8af06cee8fa67816b). Thanks!